dashboard/app/models/section.rb

```diff
@@ -61,6 +61,11 @@ def students_attributes=(params)
   end
 
   def add_student(student)
+    # TODO: generalize to future PLC sections
+    if code && code == Plc::UserCourseEnrollment::PLC_SECTION.try(:code) && !Plc::UserCourseEnrollment.exists?(user: student)
```

Comment:
Does this need to be a try? If there is no code, that's a problem because it means there is no section PRJWJN